src/js/components/agency/visualizations/StatusOfFundsChart.jsx

```diff
@@ -207,15 +207,21 @@ const StatusOfFundsChart = ({
     let tooltipName = null;
     const tooltip = (data) => {
         if (hoverData) {
-            if (data.name.length <= 33) {
-                tooltipName = data.name.length + 230;
-            }
-            else if (data.name.length > 33 && data.name.length < 66) {
-                tooltipName = data.name.length + 215;
+            if (level !== 4) {
```

Comment:
there needs to be a check for level 4 too.
level #5 and level #1 - 3 looks good.

Level #5
![Screenshot 2023-06-09 at 2 48 56 PM](https://github.com/fedspendingtransparency/usaspending-website/assets/121010/787ff507-3716-47a6-ad74-306dc2eaa869)

level #3
![Screenshot 2023-06-09 at 2 48 12 PM](https://github.com/fedspendingtransparency/usaspending-website/assets/121010/1a7837fc-315e-4329-ba58-0e16c2dd7543)

Level #4
![Screenshot 2023-06-09 at 2 47 58 PM](https://github.com/fedspendingtransparency/usaspending-website/assets/121010/a8297eea-6bd5-4b13-98b0-0f6432c738cb)

Reply:
FIXED